var/spack/repos/builtin/packages/pipx/package.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Pipx(PythonPackage):
+    """pipx is a tool to install and run Python applications in isolated environments"""
+
+    homepage = "https://pypa.github.io/pipx/"
+    pypi = "pipx/pipx-1.2.0.tar.gz"
+
+    version("1.2.0", sha256="d1908041d24d525cafebeb177efb686133d719499cb55c54f596c95add579286")
+
+    depends_on("python@3.7:", type=("build", "run"))
+
+    depends_on("py-hatchling@0.15.0:", type="build")
+
+    depends_on("py-argcomplete@1.9.4:", type=("build", "run"))
+    # FIXME: What's the OS code for Windows in general?
+    # depends_on("py-colorama@0.4.4:", type=("build", "run"), when="os=win32")
+    depends_on("py-importlib-metadata@3.3.0:", type=("build", "run"), when="^python@3.7")
+    depends_on("py-packaging@20.0:", type=("build", "run"))
+    depends_on("py-platformdirs@2.1.0:", type=("build", "run"))
```

Comment:
Where do you see this dep?

Reply:
I saw it in the main `pipx` repo ([here](https://github.com/pypa/pipx/blob/6959f93a785a92acf99705c667cc8925a31b21f3/pyproject.toml#L29)), but apparently that was added since the last release. Removed.